vcpkg/get_started/get-started-vs.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Install and use packages with vcpkg in Visual Studio
+description: Tutorial guides the user through the process of installing and using packages with vcpkg and Visual Studio.
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: operating-systems
+author: JavierMatosD
+ms.author: javiermat
+ms.topic: tutorial
+ms.date: 09/27/2023
+ms.prod: vcpkg
+#CustomerIntent: As a beginner C++ developer, I want to learn how to install and manage packages using vcpkg, so that I can easily set up and maintain C++ projects with necessary dependencies.
+---
+
+# Tutorial: Install and use packages with vcpkg
+
+This tutorial shows you how to create a C++ "Hello World" program that uses the `fmt` library with CMake, vcpkg and Visual Studio. You'll install dependencies, configure, build, and run a simple application.
+
+## Prerequisites
+
+- [Visual Studio with C++ development workload](https://visualstudio.microsoft.com/downloads/)
+- [CMake](https://cmake.org/download/)
+- [Git](https://git-scm.com/downloads)
+
+[!INCLUDE [setup-vcpkg](setup-vcpkg.md)]
+
+
+## 2 - Set up the Visual Studio project
+
+1. Create the project directory
+
+    We'll be working within a directory named `helloworld`. Ensure that this directory is a sibling of your vcpkg directory, meaning both directories should reside under the same parent directory.
+    
+    * Open Visual Studio
+    * Click on "Create a new project."
+    * Choose "Empty Project" as the template.
+    * Name your project "helloworld."
+    * Set the location to be the parent directory of your vcpkg installation.
+    * Check the box for "Place solution and project in the same directory."
+    * Click the "Create."
+  
+2. Generate a Manifest file
+
+    Create a vcpkg manifest file (`vcpkg.json`) by executing the following command in Visual Studio's built-in terminal:
+
+    ```console
+    ../vcpkg/vcpkg new --application
+    ```
+
+    The `vcpkg new` command adds a `vcpkg.json` file and a `vcpkg-configuration.json` file to your projects directory. To view the manifest file (`vcpkg.json`) toggle the "Folder view" in the solution explorer.
+
+    The `vcpkg.json` file should look like this:
+
+    ```json
+    {}
+    ```
+
+    This is your manifest file. vcpkg reads the manifest file to learn what dependencies to install and integrates with CMake to provide the dependencies required by your project. Initially, the file contains an empty JSON object; additional attributes like dependencies and supported features can be added as your project grows.
+
+    The default `vcpkg-configuration.json` file introduces [baseline](../reference/vcpkg-configuration-json.md#registry-baseline) constraints, specifying the minimum versions of dependencies that your project should use. While this allows for more granular control over your project's dependencies, a detailed discussion is beyond the scope of this tutorial.
```

Comment:
I think this wording still leaves the user confused about their state; should I keep the file or not? Instead, I think it would be better to say something like "a detailed discussion of modifying this file is beyond the scope ..." and to explicitly call out that it _should_ be added to your source control.